examples/gopherjs-humble/go/templates/partials/todo.tmpl

```diff
@@ -0,0 +1,8 @@
+<!-- <li {{ if .Completed }}class="completed"{{ end }}> -->
```

Comment:
Is this HTML comment meant to be here? What is its purpose, is it to give context on how this template is used? Or is this not a comment somehow?